boxoffice/views/order.py

```diff
@@ -152,6 +152,9 @@ def free(order):
                 line_item.discount_coupon.update_used_count()
                 db.session.add(line_item.discount_coupon)
         db.session.commit()
+        webhook_url = order.item_collection.organization.details.get('slack_webhook_url')
```

Comment:
This can actually just be `order.organization`, since order stores `organization_id`.